consensus/src/simplex/actors/voter/actor.rs

```diff
@@ -571,10 +612,12 @@ impl<
                 return Ok((GENESIS_VIEW, *self.genesis.as_ref().unwrap()));
             }
 
-            // If have notarization, return
+            // If notarized and certified, return
             let parent = self.is_notarized(cursor);
             if let Some(parent) = parent {
-                return Ok((cursor, *parent));
+                if self.is_certified(cursor) {
```

Comment:
I suppose this is the most "correct" approach (to not assume nullification if not certified).